state/controller.go

```diff
@@ -3,60 +3,10 @@
 
 package state
 
-import (
-	"github.com/juju/names/v6"
-)
-
 // Controller encapsulates state for the Juju controller as a whole,
 // as opposed to model specific functionality.
 //
 // This type is primarily used in the state.Initialize function, and
 // in the yet to be hooked up controller worker.
 type Controller struct {
```

Comment:
I think this can be removed entirely fairly easily now

Reply:
`state/initialize.go` func `Initialize` is the only reference, which I don't think is called anywhere